### PR TITLE
Fix device-tree location

### PR DESCRIPTION
**Pull request by IgorBagnucki**
Signed-off-by: Igor Bagnucki <igor.bagnucki@3mdeb.com>

### CODE REVIEW

**Comment by macpijan** [comment]
@IgorBagnucki Please use the commit message according to coreboot guidelines: https://doc.coreboot.org/tutorial/part2.html#step-4a-use-the-command-line-to-stage-and-submit-a-commit

**Comment by IgorBagnucki** [comment]
@macpijan I have fixed the commit message and added more comments to device-tree

**Comment by macpijan** [comment]
@IgorBagnucki Please take a look at other commits in the tree. It is also common to shorten the file path. For example

`src/mainboard/protectli/vault_kbl/devicetree.cb` -> `mb/protectli/vault_kbl/devicetree`

Also, there is a typo in `verbouse`

File: src/mainboard/protectli/vault_kbl/devicetree.cb [hunk 1]
```diff
@@ -184,17 +184,17 @@ chip soc/intel/skylake
...
+	# USB 2.0 enable ports 1-8 and 10, disable ports 9, 11-12
 	register "usb2_ports[0]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[1]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[2]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[3]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[4]"  = "USB2_PORT_SHORT(OC_SKIP)"	# Type-A Port
 	register "usb2_ports[5]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[6]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
-	register "usb2_ports[7]"  = "USB2_PORT_SHORT(OC_SKIP)"  # FW6: mPCIe, FW6D: mSATA slot
+	register "usb2_ports[7]"  = "USB2_PORT_SHORT(OC_SKIP)"  # FW6: mPCIe, FW6D: mSATA1 slot
```
Comment (miczyg1) [inline src/mainboard/protectli/vault_kbl/devicetree.cb:195]: no need to add the number in the port marking, there is only 1 mSATA port either way

File: src/mainboard/protectli/vault_kbl/devicetree.cb [hunk 2]
```diff
@@ -184,17 +184,17 @@ chip soc/intel/skylake
...
 	register "usb2_ports[8]"  = "USB2_PORT_EMPTY"		# Disabled
-	register "usb2_ports[9]"  = "USB2_PORT_SHORT(OC_SKIP)"  # mPCIe slot
+	register "usb2_ports[9]"  = "USB2_PORT_SHORT(OC_SKIP)"  # mPCIe slot, WIFI1 slot
```
Comment (miczyg1) [inline src/mainboard/protectli/vault_kbl/devicetree.cb:197]: no need to add the number in the port marking, there is only 1 mPCIe WiFi port either way

**Review by miczyg1 — CHANGES_REQUESTED** [review]
Let's not introduce unnecessary changes

File: src/mainboard/protectli/vault_kbl/devicetree.cb (hunk 2 above)
Comment (IgorBagnucki) [reply src/mainboard/protectli/vault_kbl/devicetree.cb]: Removed the number from the port marking

File: src/mainboard/protectli/vault_kbl/devicetree.cb (hunk 1 above)
Comment (IgorBagnucki) [reply src/mainboard/protectli/vault_kbl/devicetree.cb]: Removed the number from the port marking

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**